clang/lib/DPCT/APINamesCUB.inc

```diff
@@ -891,360 +891,382 @@ CONDITIONAL_FACTORY_ENTRY(
-                            LITERAL("false"), LITERAL("true"),
-                            ARG(4), ARG(5)))),
-                CASE(CheckArgCount(5, std::greater_equal<>(),
-                                   /* IncludeDefaultArg */ false),
-                     CALL_FACTORY_ENTRY(
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
-                            CALL("oneapi::dpl::execution::device_policy",
-                                 QUEUESTR),
-                            ARG(2), ARG(3),
-                            LITERAL("false"), LITERAL("true"),
-                            ARG(4)))),
-                OTHERWISE(
-                     CALL_FACTORY_ENTRY(
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
-                            CALL("oneapi::dpl::execution::device_policy",
-                                 QUEUESTR),
-                            ARG(2), ARG(3),
-                            LITERAL("false"), LITERAL("true")))))),
-         OTHERWISE(
-              CASE_FACTORY_ENTRY(                   
-                CASE(makeCheckAnd(
-                              CheckArgCount(8, std::greater_equal<>(),
-                                            /* IncludeDefaultArg */ false),
-                              makeCheckNot(CheckArgIsDefaultCudaStream(7))),
-                     CALL_FACTORY_ENTRY(
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
-                            CALL("oneapi::dpl::execution::device_policy",
-                                  STREAM(7)),
-                            ARG(2), ARG(3), ARG(4), LITERAL("false"), ARG(5), ARG(6)))),
-                CASE(CheckArgCount(7, std::greater_equal<>(),
-                                   /* IncludeDefaultArg */ false),
-                     CALL_FACTORY_ENTRY(
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
-                             CALL("oneapi::dpl::execution::device_policy",
-                                  QUEUESTR),
-                             ARG(2), ARG(3), ARG(4), LITERAL("false"), ARG(5), ARG(6)))),
-                CASE(CheckArgCount(6, std::greater_equal<>(),
-                                   /* IncludeDefaultArg */ false),
-                     CALL_FACTORY_ENTRY(
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
-                             CALL("oneapi::dpl::execution::device_policy",
-                                  QUEUESTR),
-                             ARG(2), ARG(3), ARG(4), LITERAL("false"), ARG(5)))),
-                OTHERWISE(
-                     CALL_FACTORY_ENTRY(
-                       "cub::DeviceRadixSort::SortKeys",
-                       CALL("dpct::sort_keys",
+    CASE(CheckCubRedundantFunctionCall(),
```

Comment:
please don't re-format inc in this PR, keep footprint as small as possible.

Reply:
done